.github/workflows/test.yml

```diff
@@ -152,3 +152,27 @@ jobs:
         required-ros-distributions: eloquent
     - run: test -f /opt/ros/eloquent/setup.sh
       name: "Check that eloquent setup.sh has been installed."
+
+  # ROS Eloquent Elusor (Windows)
+  test_eloquent_windows:
+    name: "ROS Eloquent Elusor (Windows)"
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+    - run: npm ci
+    - run: npm run build
+
+    - uses: ./ # Uses an action in the root directory
+      with:
+        required-ros-distributions: eloquent
```

Comment:
Could we make this a [matrix](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategy) such that it runs on dashing as well?

Reply:
Sure thing.